retroarch.c

```diff
@@ -1141,7 +1141,7 @@ static const camera_driver_t *camera_drivers[] = {
 
 #define AUDIO_BUFFER_FREE_SAMPLES_COUNT (8 * 1024)
 
-#define MENU_SOUND_FORMATS "ogg|mod|xm|s3m|mp3|flac"
+#define MENU_SOUND_FORMATS "ogg|mod|xm|s3m|mp3|flac|wav"
```

Comment:
Is there a specific reason `wav` was not included in this list? I did notice that the rwav library didn't like any of my `.wav` files that contained metadata.

Reply:
I can confirm the wav file won't play if any metadata is present. I wouldn't be surprised if that's why it was excluded. Easy to miss that being the cause in testing.